frontend/src/metabase/dashboard/hoc/DashboardControls.jsx

```diff
@@ -84,7 +86,12 @@ export default ComposedComponent =>
         };
         setValue("refresh", this.state.refreshPeriod);
         setValue("fullscreen", this.state.isFullscreen);
-        setValue("theme", this.state.isNightMode ? "night" : null);
+        const theme = this.state.isNightMode
```

Comment:
Only set mode to transparent when `theme` is originally `transparent` and dashboard night mode is off.